### PR TITLE
Add presubmit jobs for node-feature-discovery-operator

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Add rudimentary jobs for CI testing.

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
@marquiz

**Comment by ArangoGutierrez** [comment]
This PR enables https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/13

**Comment by ArangoGutierrez** [comment]
depends on: image-pushing: add config for nfd-operator #19141

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - ArangoGutierrez
+  - marquiz
+  - zvonkok
+approvers:
+  - ArangoGutierrez
+  - marquiz
+  - zvonkok
```
Comment (marquiz) [inline config/jobs/kubernetes-sigs/node-feature-discovery-operator/OWNERS:10]: you need to drop non-members from here

Comment (ArangoGutierrez) [reply config/jobs/kubernetes-sigs/node-feature-discovery-operator/OWNERS]: done, is just you for now, I think once this PR is on, I will apply to be a member and join the OWNERS file

**Comment by ArangoGutierrez** [comment]
Now that https://github.com/kubernetes/test-infra/pull/19141 has been merged, we can move forward here
*PING* @marquiz 
/verify-owners
/assign @spiffxp

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes-sigs/node-feature-discovery-operator:
+  - name: pull-node-feature-discovery-operator-verify
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-operator-discovery
```
Comment (marquiz) [inline config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml:7]: This should be `sig-node-node-feature-discovery`

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      testgrid-tab-name: verify
```
Comment (marquiz) [inline config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml:8]: `operator-verify`

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    spec:
+      containers:
+      - image: golang:1.14
+        command:
+        - scripts/test-infra/verify.sh
+  - name: pull-node-feature-discovery-operator-build-image
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery-operator
```
Comment (marquiz) [inline config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml:19]: `sig-node-node-feature-discovery`

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      testgrid-tab-name: build-image
```
Comment (marquiz) [inline config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml:20]: `operator-build-image`

**Review by marquiz — CHANGES_REQUESTED** [review]
Spotted some mistakes. Please double-check...

File: config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml (hunk 2 above)
Comment (ArangoGutierrez) [reply config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml:7]: woops.....

**Comment by ArangoGutierrez** [comment]
Comments addressed - @marquiz

**Review by marquiz — APPROVED** [review]
/lgtm

**Comment by ArangoGutierrez** [comment]
Can we get some :eyes: on this PR, thanks

**Review by spiffxp — APPROVED** [review]
/ok-to-test
/approve

**Pull request merged**